app/scripts/components/common/blocks/block-map.tsx

```diff
@@ -112,6 +112,8 @@ interface MapBlockProps {
   datasets: VedaDatum<DatasetData>;
   dateTime?: string;
   compareDateTime?: string;
+  compareDataSetId?: string;
+  compareLayerId?: string;
```

Comment:
nit: `compareDatasetId`, no need to capitalize the `s` but do we need to pass that in? It wont be used it looks like. datasetId is never referenced from `baseMapStaticCompareData`

Reply:
It is optional. We have to pass tha in ,if only we want to compare the datasets.